### PR TITLE
feat(app): 119 Add props to rigidbody

### PR DESCRIPTION
**Pull request by JaimeTorrealba**
Colliders props will be in another PR. I would like to discuss colliders later (for discord) @Neosoulink 

This PR comes with a way to define a set of some of the most important props for the rigidBody instance. I have a similar idea in mind for the automatics colliders and custom colliders, but I have some questions first.

### CODE REVIEW

File: src/types/rigid-body.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ export type RigidBodyType =
   | 'kinematicVelocity'
   | 'fixed'
 
+export interface enableBolean { x: boolean, y: boolean, z: boolean }
```
Comment (Neosoulink) [inline src/types/rigid-body.ts:13]: I think we can move this one to a separate type file to keep the `rigid-bosy.ts` file clean

something like `boolean.ts` or `object.ts`

**Review by Neosoulink — CHANGES_REQUESTED** [review]

File: docs/guide/how-does-it-work.md [hunk 2]
```diff
@@ -1 +1,6 @@
 # How does it work?
+
+## Caveats
+
+- Rapier set object to sleep...
+- HMR sometimes work unexpected prefer reload page (unles reactive property is set)
```
Comment (Neosoulink) [inline docs/guide/how-does-it-work.md:6]: We have to figure out how to optimize this behavior 🤔

Comment (JaimeTorrealba) [reply docs/guide/how-does-it-work.md:6]: Maybe not. I believe there's a PR (if not in events) to handle sleep and wake events.

But if not, this is the default behaviour of rapier. I think we should try to let the user decide.

In any case is not of the scope of this PR :)

Comment (Neosoulink) [reply docs/guide/how-does-it-work.md:6]: Yeah, this was just a comment. It's definitely not in scope of this PR

**Review by Neosoulink — APPROVED** [review]

**Pull request merged**